package.json

```diff
@@ -21,32 +21,32 @@
     "create-encryption-keys": "touch .env.local && cm-env setup prod && cm-env setup staging && cm-env setup dev && npm run setup-encryption",
     "test-all": "npm run lint && npm run build:staging --debug",
     "build-sitemap": "next-sitemap",
-    "prepare": "husky install"
+    "prepare": "husky"
   },
   "dependencies": {
-    "@ant-design/cssinjs": "^1.20.0",
-    "@ant-design/icons": "^5.3.7",
-    "@apollo/client": "^3.10.4",
-    "antd": "^5.17.3",
+    "@ant-design/cssinjs": "^1.22.0",
+    "@ant-design/icons": "^5.5.1",
+    "@apollo/client": "^3.11.10",
+    "antd": "^5.22.1",
     "classnames": "^2.5.1",
-    "eslint-config-next": "^14.2.3",
-    "graphql": "^16.8.1",
-    "next": "^14.2.3",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "graphql": "^16.9.0",
+    "next": "^14.2.18",
+    "react": "^18",
+    "react-dom": "^18"
```

Comment:
Why are the minor versions being removed?

Reply:
It will automatically update to the latest version of react 18, that's how it is in the next template.